build/dependencies.targets

```diff
@@ -0,0 +1,51 @@
+﻿<Project>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.AspNet.Identity.AspNetCoreCompat" Version="0.3.1-rtm-*" />
```

Comment:
How does this file work? Do we only have versions for things that change \ versions that the repo references?

Reply:
It's missing all of the Extensions packages which is why I wasn't sure how you go to this.

Reply:
I generated these files from aspnet/Universe using this target: https://github.com/aspnet/Universe/commit/a10ba83751cab0af7dc1d2a032959f5ae84824f1

This list contains all packages that will be produced in the current universe build minus any packages that come from the current repository.

The list is suboptimal. It doesn't include external dependencies (yet), and it includes Update to packages that shouldn't be used here. Consider this an minimally viable version of the list for now to get our cascading versions working.